.env.sample

```diff
@@ -5,3 +5,4 @@ AWS_SECRET_ACCESS_KEY=
 AWS_REGION=
 S3_BUCKET_NAME=
 SPARKPOST_PASSWORD=
+export APPSIGNAL_PUSH_API_KEY=
```

Comment:
I'm afraid it won't work with `export`.

Reply:
that's what they say in the instructions and we use it like that in other projects